core/src/test/java/org/mskcc/cbio/oncokb/util/FindRelevantAlterationsTest.java

```diff
@@ -53,7 +53,7 @@ public FindRelevantAlterationsTest(String hugoSymbol, String alteration, String
                 // General truncating consequence should be associated with Truncating Mutations
                 // Check splice
                 // TP53 Oncogenic Mutations does not have any information we are ready to relase
-                {"TP53", "X33_splice", null, "X33_splice, Truncating Mutations"},
+                {"TP53", "X33_splice", null, "X33_splice, Truncating Mutations, Oncogenic Mutations"},
```

Comment:
This seems that you did not run the sql script. These Oncogenic Mutations should not be there